pkg/ingester/instance_test.go

```diff
@@ -473,7 +474,7 @@ func Test_ChunkFilter(t *testing.T) {
 	require.NoError(t, err)
 	instance := newInstance(
 		&ingesterConfig, "fake", NewLimiter(overrides, NilMetrics, &ringCountMock{count: 1}, 1), loki_runtime.DefaultTenantConfigs(), noopWAL{}, NilMetrics, nil, &testFilter{})
-	ctx := context.TODO()
+	stats, ctx := stats.NewContext(context.Background())
```

Comment:
nit: May be `noopStatsContext` should work here for this test? (if we don't have constructor for it, may be create one?)

Reply:
Not sure using the real stats context is fine too.